ang/civicase-base/services/crm-api.service.js

```diff
@@ -0,0 +1,26 @@
+(function (_, angular) {
+  var module = angular.module('civicase-base');
+
+  module.service('civicaseCrmApi', function ($q, crmApi) {
+    return function (entity, action, params, message) {
+      var deferred = $q.defer();
+
+      crmApi(entity, action, params, message)
+        .then(function (result) {
+          var isError = false;
+
+          if (_.isArray(result)) {
+            _.each(result, function (response) {
+              if (response.is_error) {
+                isError = true;
+              }
+            });
```

Comment:
Can be refactored like this:

```js
isError = _.some(result, 'is_error);
```

Reply:
@reneolivo To be honest, I am not sure about this one. For a simple check, I think `if` is more readable than a custom `lodash` function.